lib/services/internalEngine/index.js

```diff
@@ -529,11 +473,12 @@ class InternalEngine {
    * @returns {Promise}
    */
   createInternalIndex () {
-    return this.client.indices.exists({index: this.index})
+    // TODO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)